src/main/resources/db/migration/V1_4__more_referral_fields.sql

```diff
@@ -0,0 +1,3 @@
+alter table referral
+    add column service_categoryid uuid,
+    add column complexity_levelid uuid;
```

Comment:
Should these two columns have the 'id' portion separated by an underscore (e.g. service_category_id) to be consistent with the complexity_level table?

Reply:
good question - these are quirks of the JPA default naming which means we don't have to specify the column name for each entity property

Reply:
OK!